test/e2e/specs/editor/blocks/columns.spec.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	getBlockFunction: async ( { page }, use ) => {
+		await use( new GetAllBlockInserterItem( { page } ) );
+	},
+} );
+
+test.describe( 'Columns Test', () => {
```

Comment:
```suggestion
test.describe( 'Columns', () => {
```

Let's use the same names as in the original tests.